src/main/java/github/weichware10/toolbox/gui/dialogs/ConfirmDialog.java

```diff
@@ -0,0 +1,65 @@
+package github.weichware10.toolbox.gui.dialogs;
+
+import java.io.IOException;
+import java.util.Optional;
+
+import javafx.fxml.FXMLLoader;
```

Comment:
⚠️ **[checkstyle]** <com.puppycrawl.tools.checkstyle.checks.imports.CustomImportOrderCheck> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Extra separation in import group before 'javafx.fxml.FXMLLoader'